cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -3855,6 +3855,13 @@ def _get_initial_annotation(annotation_format):
                 annotations["shapes"] = rectangle_shapes_wo_attrs \
                                       + polygon_shapes_wo_attrs
 
+            elif annotation_format == "WiderFace 1.0":
+                annotations["tags"] = tags_wo_attrs
```

Comment:
This format should support boxes and few attributes. How about including them in tests and in documentation?